specifications/FiniteMonotonic/CRDT.tla

```diff
@@ -0,0 +1,41 @@
+------------------------------- MODULE CRDT ---------------------------------
+
+EXTENDS Naturals
+
+CONSTANT Node
+
+VARIABLE counter
+
+TypeOK == counter \in [Node -> [Node -> Nat]]
+
+Safety == \A n, o \in Node : counter[n][n] >= counter[o][n]
+
+Monotonic == \A n, o \in Node : counter'[n][o] >= counter[n][o]
+
+Monotonicity == [][Monotonic]_counter
+
+Convergence == \A n, o \in Node : counter[n] = counter[o]
```

Comment:
This is not a liveness property, but a state predicate on the initial state.

Reply:
This just defines what it means for the system to have converged, there's a liveness property in `Finitize_CRDT.tla` with the definition `Liveness == converge ~> S!Convergence`.

Reply:
The blog post stats that this is a liveness property:

> Finally, the big one: this is an eventually-consistent system, so we want to validate its eventual consistency as a liveness property! We want to say it is always the case that eventually the vectors on all nodes will have the same value. Let’s define this goal state as:

https://github.com/tlaplus/Examples/pull/97/files#diff-9de53831fa4828bac3abea3e31ce96799638174ee08488d65cb5726b09113c3bR151-R155

If anything, the blog post has to be corrected.  I presume the desired property would be `\A n, o \in Node: []<>(counter[n] = counter[o])`, combined with asserting fairness over `Gossip`.

Reply:
It could be misread but it says it defines the "goal state" as that, not the liveness property generally

Reply:
"goal state" is rather ambiguous in a non-terminating system?  At any rate, it seems easy to make less ambiguous.